static/app/views/alerts/rules/metric/triggers/actionsPanel/index.tsx

```diff
@@ -321,9 +321,6 @@ class ActionsPanel extends PureComponent<Props> {
       value: getActionUniqueKey(availableAction),
       label: getFullActionTitle(availableAction),
     }));
-    const hasPriorityFlag = organization.features.includes(
-      'integrations-custom-alert-priorities'
```

Comment:
Found a few other places using this:
- src/sentry/integrations/pagerduty/actions/notification.py
- tests/sentry/integrations/pagerduty/test_notify_action.py

can we remove those cases as well and then link to any cleanup code PRs to remove the flag? 🙏

Reply:
yep i need to split the PRs cause of FE/BE. i am about to click to make the other PR